src/main/java/frc/robot/utils/SimmableCANSparkMax.java

```diff
@@ -125,7 +125,7 @@ public double getVelocity() {
         }
         return m_encoderCount.get();
```

Comment:
This seems wrong too.  Position and Velocity should be different things.

Reply:
Looks like I just copy-pasted the code from encoderCount and updated the
string without updating any of the actual code. Oops.